common/equipment.go

```diff
@@ -47,7 +47,8 @@ const (
 	EqP250         EquipmentElement = 3
 	EqDeagle       EquipmentElement = 4
 	EqFiveSeven    EquipmentElement = 5
-	EqDualBarettas EquipmentElement = 6
+	EqDualBerettas EquipmentElement = 6
+	EqDualBarettas EquipmentElement = 6 // Misspelled constant for compatibility
```

Comment:
```suggestion
	EqDualBarettas EquipmentElement = 6 // Deprecated, use EqDualBerettas instead (spelling error)
```

This will visually mark the old constant as deprecated and may be detected by some linters or other tools